pkgs/development/compilers/cudatoolkit/common.nix

```
@@ -291,6 +291,10 @@ backendStdenv.mkDerivation rec {
   '' + lib.optionalString (lib.versionOlder version "8.0") ''
     # Hack to fix building against recent Glibc/GCC.
     echo "NIX_CFLAGS_COMPILE+=' -D_FORCE_INLINES'" >> $out/nix-support/setup-hook
+  ''
+  # 11.8 includes a broken symlink, include/include, pointing to targets/x86_64-linux/include
+  + lib.optionalString (lib.versionAtLeast version "11.8" && (lib.versionOlder version "12.0")) ''
```

```suggestion
  + lib.optionalString (lib.versions.majorMinor version == "11.8") ''
```
If this only affects 11.8, I propose we simplify a bit

---

Yes, I somehow forgot that they're strings and we can compare them directly.

I'll submit a separate PR for `cudatoolkit` because it really is unrelated to the CUDNN bump.